aptos-move/framework/aptos-stdlib/sources/pool_u64.spec.move

```diff
@@ -0,0 +1,166 @@
+spec aptos_std::pool_u64 {
```

Comment:
Can you add comments here explaining the invariants being checked? Just short one sentences would already immensely increase readability

Reply:
That's a good point. I added some comments for the invariants.